src/components/ApplicationPrompt.astro

```diff
@@ -37,7 +37,7 @@ const regionList = listFormatter.format(cohortRegions);
 ---
 
 <p class="u-text-large">
-	Our next cohort will kick off on {formattedKickoff}. We will run teams in
+	Our next cohort will kick off on {formattedKickoff}. We plan to run teams in
```

Comment:
```suggestion
	Our next cohort will kick off on {formattedKickoff}. We plan to run teams in the
```

[puts on language pedant hat] I feel like the sentence needs a definite article here, but I'm fine omitting it if you'd prefer.

Reply:
I checked with my partner who’s a professional book editor and she thinks it’s fine without the “the”?

Reply:
Fair enough!